app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,18 @@
+class RentsController < ApplicationController
+  include Wor::Paginate
+
+  def create
+    @rent = Rent.create(rent_create_params)
```

Comment:
Does the rent have any validations? If it has and you receive invalid params, you are not  saying to the frontend that you created a new Rent, and it didn't happen.

Reply:
What I'm trying to say here is that you should handle the model validation errors, and display those to the frontend